### PR TITLE
AccountPosition adding missing 4 fields

### PR DESCRIPTION
**Pull request by straddler**
AccountPosition added 4 fields that were missing. These are rather important to have as they are giving useful information about your futures open positions, something that was missing for a long time from Binance it seems.

https://binance-docs.github.io/apidocs/futures/en/#account-information-v2-user_data

### CODE REVIEW

**Comment by straddler** [comment]
@adshao @afenrir  please if possible check this out so it can be merged and used via import. 

Thanks.

File: v2/futures/account_service.go [hunk 1]
```diff
@@ -106,4 +106,8 @@ type AccountPosition struct {
 	PositionInitialMargin  string `json:"positionInitialMargin"`
 	Symbol                 string `json:"symbol"`
 	UnrealizedProfit       string `json:"unrealizedProfit"`
+	EntryPrice             string `json:"entryPrice"`
+	MaxNotional            string `json:"maxNotional"`
+	PositionSide           string `json:"positionSide"`
```
Comment (afenrir) [inline v2/futures/account_service.go:111]: PositionSide should be of type PositionSideType 

https://github.com/adshao/go-binance/blob/8754a53fcf6a850b3f3f93c220174c9648ec8166/v2/futures/client.go#L25

**Review by afenrir — CHANGES_REQUESTED** [review]
Thanks for the PR, @straddler. I have requested one change. Other than that, it looks good to me.

File: v2/futures/account_service.go (hunk 1 above)
Comment (straddler) [reply v2/futures/account_service.go]: Great, I have made the change. Thanks for the review!

**Review by adshao — APPROVED** [review]

**Comment by adshao** [comment]
thanks @straddler

**Pull request merged**